group-owner.html

```diff
@@ -249,9 +263,7 @@
 
 <p>Any data imported by a user will be owned by them and will be in a group of which they are a member. For details on importing see <a href="http://help.openmicroscopy.org/importing-data-5.html" target="_blank" class="external-url">Importing Data</a>.
 
-<p>Administrators can import data to any group for any user i.e. the data will belong to that user.</p>
-
-<p>Similarly a Group Owner can import for other members of their group.</p>
+<p>Group Owners and Administrators, who are members of the group, can import data for any user in the group i.e. the data will belong to that user.</p>
```

Comment:
Sorry to be a pain. This sentence is now accurate for Insight. On the CLI, you can, being an admin, login as the (other) user himself, meaning you, being an admin do not have to be the member of the target group (only the user you are getting into like this has to be the member of the group of course). Really appreciate this is hard to explain in a succint form and not be confusing... There is no streamlining between CLI and Insight on this point.